app/models/returns/credits/CreditSummaryRow.scala

```diff
@@ -25,23 +25,24 @@ import uk.gov.hmrc.govukfrontend.views.viewmodels.table.TableRow
 case class CreditSummaryRow(
   label: String,
   value: String,
-  actions: Seq[ActionItem] = Seq.empty,
-  isActionColumnHidden: Boolean = false
+  actions: Seq[ActionItem] = Seq.empty
 ) {
 
   def createContent(createActionsContent: Seq[ActionItem] => Html): Seq[TableRow] = {
-    val alignRight = if (isActionColumnHidden) "govuk-table__cell--numeric" else ""  
```

Comment:
Does this break the layout on big cya?

Reply:
ok, thanks - will look in to it

Reply:
big cya appears to be working ok on partridge, looks like this branch adds a hidden extra cell on each row?

Reply:
will take a nother look